azure/scope/cluster.go

```diff
@@ -679,20 +679,7 @@ func (s *ClusterScope) PatchObject(ctx context.Context) error {
 	ctx, _, done := tele.StartSpanWithLogger(ctx, "scope.ClusterScope.PatchObject")
 	defer done()
 
-	conditions.SetSummary(s.AzureCluster,
-		conditions.WithConditions(
-			infrav1.ResourceGroupReadyCondition,
-			infrav1.RouteTablesReadyCondition,
-			infrav1.NetworkInfrastructureReadyCondition,
-			infrav1.VnetPeeringReadyCondition,
-			infrav1.DisksReadyCondition,
-			infrav1.NATGatewaysReadyCondition,
-			infrav1.LoadBalancersReadyCondition,
-			infrav1.BastionHostReadyCondition,
```

Comment:
I don't see the bastion service being modified, did that one also need to be updated?

Reply:
looks like there are 10+3 conditions being removed here and in machine.go but 10 services getting updated

Reply:
Oh yeah I think I missed those on accident, thanks for noticing!